mlir-tensorrt/tensorrt/lib/TensorRT/Transforms/ExpandOps.cpp

```diff
@@ -297,21 +299,31 @@ struct TileLikeBroadcastToSlice : public OpRewritePattern<BroadcastOp> {
       input = reshapeBroadcastInput(rewriter, op.getLoc(), op);
     TensorType inputType = input.getType();
 
-    SmallVector<int32_t> start(inputType.getRank(), 0);
-    SmallVector<int32_t> stride(start.size(), 1);
+    // Create static_start and static_stride attributes
+    auto static_start = rewriter.getDenseI32ArrayAttr(
```

Comment:
Why do we need to explicitly convert to `DenseI32ArrayAttr`? It looks like the previous code did not require that.

Reply:
The code switches to using another way of constructing `SliceOp` to make it consistent with dynamic shapes code path (check the various `build` definitions in `TensorRTOps.h.inc`.